CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [2.0.1-next.1](https://github.com/warp-ds/icons/compare/v2.0.0...v2.0.1-next.1) (2024-04-15)
+
+
+### Bug Fixes
+
+* new translations from Crowdin ([#103](https://github.com/warp-ds/icons/issues/103)) ([e991690](https://github.com/warp-ds/icons/commit/e991690680f9b8cfe7b20d42a9e6db6715fecbbe))
+
```

Comment:
The autovex merge was missing the fix: prefix so it's not included in this changelog...
![image](https://github.com/warp-ds/icons/assets/7531505/447a074b-5923-47a9-9c59-199a00fa6a57)

Reply:
Yes, it was my bad because I missed that the squash-merge used an old PR title, which didn't have the prefix. I should've mentioned it in the PR description. That's also the reason why I added a `fix:` prefix to the crowdin PR - I needed to trigger a release. So now the changelog is a bit messed up, true, but it's reflecting the release notes in [the releases page](https://github.com/warp-ds/icons/releases). I don't know if there's an easy way to update both changelog and release notes manually without breaking automated releases for next time (I believe we tried that once with Anna and it hit us back upon the next release).